policyengine_us/variables/gov/states/mt/tax/income/mt_income_tax_before_non_refundable_credits_joint.py

```diff
@@ -23,10 +33,11 @@ def formula(tax_unit, period, parameters):
                 filing_status == status.SURVIVING_SPOUSE,
             ],
             [
-                p.single.calc(income),
-                p.joint.calc(income),
-                p.head_of_household.calc(income),
-                p.separate.calc(income),
-                p.widow.calc(income),
+                p.single.calc(taxable_income),
```

Comment:
in this case, do we tax long term capital gains twice?

Reply:
I think not, in the first if clause, which is after 2024, the `taxable_income `will exclude the `long_term_capital_gains` and if it's before 2024 in the else clause, we'll calculate all the `taxable_income`. Is this correct?